types.go

```diff
@@ -21,6 +22,9 @@ const (
 )
 
 func (e EvaluationType) String() string {
+	if e < 0 || int(e) >= len(evaluationStrings) {
+		return fmt.Sprintf("EvaluationType(%d)", e)
```

Comment:
Yes, it works, because 0 is an untyped integer constant that can be compared or assigned to any compatible integer variable.